resources/lang/pl/store.php

```diff
@@ -24,7 +24,16 @@
     ],
 
     'checkout' => [
+        'cart_problems' => 'Ah, wystąpił problem z twoim koszykiem!',
+        'cart_problems_edit' => 'Naciśnij tutaj, aby go zedytować',
```

Comment:
edytować

Reply:
Kliknij (i guess?)

Reply:
Raczej zedytować bo to nie jest ciągła czynność

Reply:
fair, I guess, niech ktoś jeszcze się wypowie